### DISCUSSION TITLE
Saludar

### DISCUSSION THREAD

**Discussion by frankinkua** [inkua/information-technology-it-workspace#3] [General]
Soy Frank de Frankfurt. Director de área de Tecnología de InkuA. Le doy la bienvenida a quienes quieran  introducirse y saludar en este thread.

**Comment by LucasAlvarez99** [comment]
Hola soy lucas uno de los nuevos integrantes de inkua en el area de sistemas perteneciente al grupo de desarrollo b

**Comment by Valen1611** [comment]
Buenas, soy Valentin del area de sistemas!

**Comment by Igris-1** [comment]
Holaa soy Nico, nuevo en Inkua e integrante del área de sistemas, departamento B.

**Comment by manuel14mds** [comment]
Hola muy buenas, soy Manuel Florez, tambien soy nuevo . Ayer Cielo Montero se comunicó conmigo y hoy hice el pull request el pt1.

**Comment by VioleHours** [comment]
Hola buenas! me paso Paula Toledo el link para que subiera lo del TP1, pero no estoy como miembro, me podrían agregar por favor? Gracias.

**Reply by manuel14mds** [reply]
Hola Violeta! muy buenas!
Para colaborar debes hacer un Fork del proyecto. En el TP1 hay un enlace a un video que explica un paso a paso. 
Te paso otro video y si sigues teniendo problemas respondes este mensaje y lo vemos.
https://drive.google.com/file/d/1eydeAOTsdM9_RMVwy2s5OtootmB_nFH_/view?usp=sharing
